src/main/java/eu/dissco/orchestration/backend/controller/RestResponseEntityExceptionHandler.java

```diff
@@ -24,10 +25,16 @@ public ResponseEntity<String> jsonProcessingException(JsonProcessingException e)
     return ResponseEntity.status(HttpStatus.UNPROCESSABLE_ENTITY).body(e.getMessage());
   }
 
-  @ResponseStatus(HttpStatus.BAD_REQUEST)
+  @ResponseStatus(HttpStatus.CONFLICT)
   @ExceptionHandler(IllegalArgumentException.class)
   public ResponseEntity<String> illegalArgumentException(IllegalArgumentException e) {
     return ResponseEntity.status(HttpStatus.CONFLICT).body(e.getMessage());
   }
 
+  @ResponseStatus(HttpStatus.INTERNAL_SERVER_ERROR)
```

Comment:
Aren't we not supposed to return 5xx errors? Would Unprocessable Entity be more appropriate?

Reply:
Oef yes I know, really looked for something else but the user can't really do anything if our Kafka is down and the message cannot be published. I will take another look, see if we can do things differently. We could also just no catch this exception and let the system throw it. That way it will also be a 500 but we didn't expect it to be thrown

Reply:
I could do BAD_REQUEST but really we already processed the request and it was accepted...